source/Nevermore.IntegrationTests/Chaos/ChaosSqlCommand.cs

```diff
@@ -20,7 +21,12 @@ public ChaosSqlCommand(DbCommand wrappedCommand, double chaosFactor)
         void MakeSomeChaos()
         {
             if (Debugger.IsAttached) return; // No chaos when debugging thanks!
-            if (ChaosGenerator.NextDouble() < chaosFactor) throw new TimeoutException("You made the chaos monkey angry...");
+            if (ChaosGenerator.NextDouble() < chaosFactor)
+            {
+                if (ChaosGenerator.NextDouble() < closeConnectionChaosFactor)
+                    wrappedCommand.Connection?.Close();
```

Comment:
When the chaos monkey kicks in there is a 50% of chance of dropping the connection.